<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GomokuClass</class>
 <widget class="QWidget" name="GomokuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>0</y>
     <width>160</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btnSkipTurn">
      <property name="text">
       <string>Skip turn</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSwapMode">
      <property name="text">
       <string>Swap Mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSwap2Mode">
      <property name="text">
       <string>Swap2 mode</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnResetGame">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Gomoku.qrc"/>
 </resources>
 <connections/>
</ui>
